onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -2015,6 +2015,22 @@ def _where_input_wrangler(
         enabled_if=version_utils.torch_older_than("2.1"),
         reason="The operator is not supported in older version.",
     ),
+    TorchLibOpInfo(
+        "ops.aten._scaled_dot_product_efficient_attention",
+        nn_ops.aten__scaled_dot_product_efficient_attention,
+        trace_only=True,
+        tolerance={torch.float32: (3e-4, 1.5e-5)},
+        # Output[0] is OK, but other outputs just have the same shape with zero values
```

Comment:
Use the other compare option instead

Reply:
https://github.com/microsoft/onnxscript/blob/d8eb6b58fd5e63d46fdcd4e220ab9e6d991c5de7/onnxscript/tests/function_libs/torch_lib/ops_test_data.py#L1149C16-L1149C16

Reply:
compare_shape_only_for_output

Reply:
Done